### PR TITLE
Issue #13345: Enabled UnnecessarySemicolonInEnumerationCheckExamplesTest

### PR DESCRIPTION
**Pull request by kkoutsilis**
Part of #13345
Enabled UnnecessarySemicolonInEnumerationCheckExamplesTest

To honor the rule: `Only comments are changing in content from Example1 to Example2 and so on....`, removed Example2.txt and moved ok example to Example1. 
Also had to add a suppression to not raise `"Example test input must declare exactly one class matching ''Example\d+''"` error.

### CODE REVIEW

File: src/xdocs/checks/coding/unnecessarysemicoloninenumeration.xml [hunk 1]
```diff
@@ -31,35 +31,32 @@
...
+
+enum Six {
     A,
     B,
-    ; // required &quot;;&quot;, no violation
-    Normal(){}
-}
-enum NoSemicolon {
-    A, B // only enum constants, no semicolon required
+    ; // ok, enum body contains constructor.
```
Comment (romani) [inline src/xdocs/checks/coding/unnecessarysemicoloninenumeration.xml:58]: Please restore this case without violations, to show users what vit good code according to this Check

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/checks/coding/unnecessarysemicoloninenumeration.xml (hunk 1 above)
Comment (kkoutsilis) [reply src/xdocs/checks/coding/unnecessarysemicoloninenumeration.xml]: Restored

**Comment by romani** [comment]
Github, generate website

**Review by romani — APPROVED** [review]
ok to merge if CI pass.

Thanks a lot !!!

**Comment by kkoutsilis** [comment]
> ok to merge if CI pass.
> 
> Thanks a lot !!!

Happy to help!

**Comment by romani** [comment]
please move to next level - https://github.com/checkstyle/checkstyle/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+third+issue%22

**Pull request merged**